src/ng/rootScope.js

```diff
@@ -418,8 +418,9 @@ function $RootScopeProvider() {
        *    expression in `watchExpressions` changes
        *    The `newValues` array contains the current values of the `watchExpressions`, with the indexes matching
        *    those of `watchExpression`
-       *    and the `oldValues` array contains the previous values of the `watchExpressions`, with the indexes matching
-       *    those of `watchExpression`
+       *    and the `oldValues` array contains the value of each `watchExpression` before the last change (it 
```

Comment:
I agree it is not perfect. It is pretty tricky to describe this behavior concisely and clearly. New attempt:

and the `oldValues` array contains the value of each `watchExpression` before it was last changed (even if that change did not happen in this digest cycle).

Happy for any suggestion with regards to wording - as long as we mention this for me unexpected behavior explicitly. If you can tell which part exactly find hard to understand, I will try to improve that specifically.